R/metadata.R

```diff
@@ -27,15 +27,23 @@
 #' @slot load.rate.units character. The units in which load (flux) is reported 
 #'   by predict.loadModel().
 #' @slot station character. A description of the sampling station or site.
+#' @slot wq.sta.id character Station ID for the constituent
+#' @slot station.lat numeric Station latitude
+#' @slot station.lon numeric Station longitude
+#' @slot disch.sta.id character Station ID for discharge
+#' @slot disch.basin.area numeric Drainage basin area for discharge station
+#' @slot wq.basin.area numeric Drainage basin area for water quality station
+#' @slot consti.name character Consituent long name
 #' @slot custom ANY. Empty by default, but may be modified to store any 
 #'   additional data the user wants to track.
 #' @importFrom methods setClass
 #' @exportClass metadata
 setClass(
   "metadata",
   slots=c(
-    # Column names
+    # Column names (except consti.name)
```

Comment:
but now `consti.name` is included here, right? and it's just the metadata() function argument names that have to stay out of order for backwards compatibility?